server/controllers/mainController.js

```diff
@@ -1,3 +1,41 @@
-exports.novelsPage = (req, res) => {
-  res.render("../views/main");
+const mongoose = require('mongoose')
+const Novel = mongoose.model('Novel')
+const path = require('path')
+const fs = require('fs')
+const util = require('util')
+const year = require('year')
+
+exports.mainPage = (req, res) => {
+	const copyrightYear = year()
+	res.render('../views/index', { copyrightYear })
+}
+
+exports.novelsPage = async (req, res) => {
+	try {
+		const curChapterNr = parseInt(req.params.chapterNr)
```

Comment:
Even though it's longer, I think using `current` instead of `cur` would be easier to understand :)

Reply:
updated